### ISSUE TITLE
Accessibility at 100

### ISSUE THREAD

**Issue by brycemara** [brycemara/overlook#15]
- Accessibility audits should be at 100%
- A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
- Application should be fully responsive for mobile - tablet - desktop
